server/portal/apps/accounts/migrations/0001_initial.py

```diff
@@ -32,8 +32,6 @@ class Migration(migrations.Migration):
             name='PortalProfile',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('ethnicity', models.CharField(max_length=255)),
-                ('gender', models.CharField(max_length=255)),
```

Comment:
We want to keep old migrations unaltered, and rather all our future changes can be handled by additional migrations